source/manual/debug-underperforming-search.html.md

```diff
@@ -4,20 +4,53 @@ title: How to debug underperforming search
 section: Monitoring
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2019-12-05
+last_reviewed_on: 2020-02-04
 review_in: 3 months
 ---
 
-Search is one of the more load-sensitive parts of GOV.UK, as it can't be cached as effectively as more static pages.  There are two significant components involved in search: the search-api application, and the AWS-managed Elasticsearch cluster powering it.
+Search is one of the more load-sensitive parts of GOV.UK, as it can't
+be cached as effectively as more static pages.  There are two
+significant components involved in search: the search-api application,
+and the AWS-managed Elasticsearch cluster powering it.
 
 Useful metrics to look at are:
 
-- Request duration from finder-frontend to search-api and request duration from search-api to Elasticsearch, both on the [Search API / Elasticsearch SLIs dashboard](https://grafana.blue.production.govuk.digital/dashboard/file/search_api_elasticsearch.json?orgId=1).
+- Request duration from finder-frontend to search-api, on the [finder-frontend app dashboard](https://grafana.blue.production.govuk.digital/dashboard/file/finder-frontend.json?refresh=5s&orgId=1)
 
-    If the search-api to Elasticsearch duration has increased, then there may be a capacity issue with Elasticsearch.  If only the finder-frontend to search-api duration has increased, then there may be a capacity issue with search-api.
+    If this has increased then there may be a capacity issue with
+    search-api.
+
+- Request duration from search-api to Elasticsearch and SageMaker, on the [search-api app dashboard](https://grafana.blue.production.govuk.digital/dashboard/file/search-api.json?refresh=5s&orgId=1)
+
+    See the "`<thing>` req count vs latency" graphs:
+
+    - **Reranker:** if this has increased, queries sorted by relevance
+      (keyword searches) will be slower.  This could indicate a
+      performance issue with SageMaker.
+
+    - **Search:** if this has increased, all queries will be slower.
+      This could indicate a performance issue with Elasticsearch.
+
+    - **Completion suggestion:** if this has increased, autocomplete
+      suggestions on finder-frontend pages will be slower.  Other
+      search-powered pages, like taxon pages, would not be affected.
+      This could indicate a performance issue with Elasticsearch.
+
+    - **Spelling suggestion:** if this has increased, finder-frontend
+      pages will be slower.  Other search-powered pages, like taxon
+      pages, would not be affected.  This could indicate a performance
+      issue with Elasticsearch.
```

Comment:
I really like this section, thanks for putting it in. Literally the first written down documentation I've seen on what these mean.